guidelines/guidelines.php

```diff
@@ -7,39 +8,49 @@
-    </li>
-    <li>
-        You may not list servers which serve for the sole purpose of redirecting or advertising other servers or have no content (To our discretion).
-    </li>
-    <li>
-        You may not reward or force users to do actions in DISBOARD. For instance, you may not reward your users for posting a nice review on a server or force them to bump a server.
-    </li>
-    <li>
-        You may not create multiple Discord accounts to submit multiple reviews. Please just post 1 review per person.
-    </li>
-    <li>
-        All servers which are mainly based on NSFW (sexual content to our discretion) must be marked as "NSFW" in DISBOARD.
-    </li>
-</ul>
+<div>
+    <h2>General Guidelines</h2>
+    <p>Below are a set of guidelines that applies to the entirity of our service.</p>
+    <ul>
+        <li>You may not violate any laws or regulations in your country of residence or promote the violation of them.</li>
+        <li>You may not automate any form of actions on our service</li>
+        <li>You may not use foul and/or sexual language on our service (to our discretion), in meta (content posted by users)</li>
+        <li>
+            You may not reward, harass or force users to interact with our service (e.g. bumping or reviewing a server), including but not limited to:
+            <ul>
+                <li>Any form of virtual or non-virtual payment.</li>
+                <li>Leaderboards (or other forms of explicit tracking of users' actions on our service)</li>
+                <li>Roles (or other benefits over users not interacting with our service)</li>
+                <li>Incentives (promises for a future event)</li>
+            </ul>
+        </li>
+    </ul>
+</div>
+<div>
+    <h2>For Servers</h2>
+    <p>Below are a set of guidelines that applies to servers listed on our service.</p>
+    <ul>
+        <li>You may not list servers which serve the sole purpose of redirecting or advertising other servers, or that have no content within them (to our discretion).</li>
+        <li>You may not violate the <a href="https://discord.com/guidelines" target="_blank">Discord Community Guidelines</a>.</li>
+        <li>You may not list servers which are designed to mislead users, or that contains meta (title, description, icon, tags or other content on your listing) that is irrelevant to the server itself.</li>
+        <li>All servers which are mainly based on NSFW content (to our discretion) must be marked as "NSFW" on the DISBOARD site.</li>
+    </ul>
+</div>
+<div>
+    <h2>For Reviews</h2>
+    <p>Below are a set of guidelines that applies to reviews posted of listings on our service.</p>
+    <ul>
+        <li>You may not utilise multiple Discord accounts to submit multiple reviews on a listing.</li>
+        <li>You may not post reviews that are harmful or defame a user or server's reputation.</li>
```

Comment:
either (to our discretion) or specify, so for now we don't want them to accuse ToS violations and stuff, but this sounds like it can be spun around to cover all sorts of "false claims"